models/src/test/scala/coop/rchain/models/SortedParMapSpec.scala

```diff
@@ -32,7 +32,7 @@ class SortedParMapSpec extends FlatSpec with PropertyChecks with Matchers {
   )
 
   private def roundTripTest(parMap: SortedParMap): Assertion =
-    EMap.parseFrom(serializeEMap(parMap)) should ===(EMap(parMap.sortedMap.map(toKVpair).toSeq))
+    EMap.parseFrom(serializeEMap(parMap)) should ===(EMap(parMap.sortedList.map(toKVpair)))
```

Comment:
Sorry for the avalanche of changes, but...

serializeEMap is defined in this test so it isn't really testing a concept from production code. Instead, I believe the definition of that method should be using the ParMapTypMapper to convert the ParMap to EMap and do `toByteArray` from there. Good to merge after that change :)

Reply:
The methods in `ParMapTypeMapper` only work on `ParMap` while we want to test `SortedParMap` here.